dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/blob.properties

```diff
@@ -12,6 +12,9 @@ implementation=objectstorage
 # Optional, defaults to 32768 bytes (32KB), must be positive
 hybrid.size.threshold=32768
 
+# A option to store data both in cassandra and objectstorage for in case of rough backup.
+hybrid.duplicate.writes=true
```

Comment:
Why do we put a default value to true here while it is false in the code reading configuration?
Please also indicate the default value if ommited in the comment above, as well as weither this is optional or not.